gradle/build.versions.toml

```diff
@@ -0,0 +1,42 @@
+[versions]
+asm = "9.3"
+jackson = "2.13.2"
+junit5 = "5.7.1"
+spock = "2.1-groovy-3.0"
+
+[libraries]
+ant = "org.apache.ant:ant:1.10.8"
+apache-compress = "org.apache.commons:commons-compress:1.21"
+apache-rat = "org.apache.rat:apache-rat:0.11"
+asm = { group = "org.ow2.asm", name="asm", version.ref="asm" }
+asm-tree = { group = "org.ow2.asm", name="asm-tree", version.ref="asm" }
+bytebuddy = "net.bytebuddy:byte-buddy:1.12.9"
+checkstyle = "com.puppycrawl.tools:checkstyle:9.3"
+commons-codec = "commons-codec:commons-codec:1.11"
+commmons-io = "commons-io:commons-io:2.2"
+docker-compose = "com.avast.gradle:gradle-docker-compose-plugin:0.14.13"
+forbiddenApis = "de.thetaphi:forbiddenapis:3.2"
+hamcrest = "org.hamcrest:hamcrest:2.1"
+httpcore = "org.apache.httpcomponents:httpcore:4.4.12"
+httpclient = "org.apache.httpcomponents:httpclient:4.5.10"
+idea-ext = "gradle.plugin.org.jetbrains.gradle.plugin.idea-ext:gradle-idea-ext:1.1.1"
+json-schema-validator = "com.networknt:json-schema-validator:1.0.49"
+jackson-dataformat-yaml = { group = "com.fasterxml.jackson.dataformat", name="jackson-dataformat-yaml", version.ref="jackson" }
+jackson-platform = { group = "com.fasterxml.jackson", name="jackson-bom", version.ref="jackson" }
+jna = "net.java.dev.jna:jna:5.10.0"
+junit = "junit:junit:4.12"
+junit5-platform = { group = "org.junit", name="junit-bom", version.ref="junit5" }
+junit5-jupiter = "org.junit.jupiter:junit-jupiter:5.8.1"
+junit5-platform-launcher = "org.junit.platform:junit-platform-launcher:1.8.0"
+junit5-vintage = "org.junit.vintage:junit-vintage-engine:5.8.1"
+maven-model = "org.apache.maven:maven-model:3.6.2"
+mockito-core = "org.mockito:mockito-core:1.9.5"
+nebula-info = "com.netflix.nebula:gradle-info-plugin:11.3.3"
+randomized-runner = "com.carrotsearch.randomizedtesting:randomizedtesting-runner:2.7.7"
```

Comment:
I think we can ditch this altogether. We only reference this class a couple of places for `assumeFalse()` which we can easily replace with `Assumptions` from JUnit.

Reply:
GradleUnitTestCase and BaseTestCase still rely on plenty of that randomizedtesting stuff and we still have plenty of test classes based on those in build-tools and build-tools-internal

Reply:
We should look to ditch those if we can. Not sure how much of a refactoring effort that would be.